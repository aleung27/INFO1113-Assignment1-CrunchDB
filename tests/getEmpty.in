GET a
SET a 3
GET a
GET b
SET a 0 9 8 7 6 5 4 3
GET a
DEL a
GET a
SET a 1
PURGE a
GET a
