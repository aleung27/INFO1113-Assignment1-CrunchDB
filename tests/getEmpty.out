> no such key

> ok

> [3]

> no such key

> ok

> [0 9 8 7 6 5 4 3]

> ok

> no such key

> ok

> ok

> no such key

